src/compute.cpp

```diff
@@ -1,876 +1,420 @@
-const double thercoCO2P3000T40 = 17.87; //At 500 psi
-const double thercoCO2P3000T60 = 19.52;
-const double thercoCO2P3000T80 = 21.11;
-
-const double thercoButP500Tneg20 = 15.9;
-const double thercoButP500T0 = 18.3;
-const double thercoButP500T20 = 20.5;
-const double thercoButP500T40 = 22.9; //At 500 psi
-const double thercoButP500T60 = 25.5;
-const double thercoButP500T80 = 27;
-
-const double thercoButP1000Tneg20 = 15.9;
-const double thercoButP1000T0 = 18.3;
-const double thercoButP1000T20 = 20.5;
-const double thercoButP1000T40 = 22.9; //At 1000 psi
-const double thercoButP1000T60 = 25.5;
-const double thercoButP1000T80 = 27;
-
-const double thercoButP1500Tneg20 = 15.9;
-const double thercoButP1500T0 = 18.3;
-const double thercoButP1500T20 = 20.5;
-const double thercoButP1500T40 = 22.9; //At 1500 psi
-const double thercoButP1500T60 = 25.5;
-const double thercoButP1500T80 = 27;
-
-const double thercoButP2000Tneg20 = 15.9;
-const double thercoButP2000T0 = 18.3;
-const double thercoButP2000T20 = 20.5;
-const double thercoButP2000T40 = 22.9; //At 2000 psi
-const double thercoButP2000T60 = 25.5;
-const double thercoButP2000T80 = 27;
-
-const double thercoButP2500Tneg20 = 15.9;
-const double thercoButP2500T0 = 18.3;
-const double thercoButP2500T20 = 20.5;
-const double thercoButP2500T40 = 22.9; //At 2500 psi
-const double thercoButP2500T60 = 25.5;
-const double thercoButP2500T80 = 27;
-
-const double thercoButP3000Tneg20 = 15.9;
-const double thercoButP3000T0 = 18.3;
-const double thercoButP3000T20 = 20.5;
-const double thercoButP3000T40 = 22.9; //At 3000 psi
-const double thercoButP3000T60 = 25.5;
-const double thercoButP3000T80 = 27;
-
-const double texasPipelineMethane = 0.96;
-const double texasPipelineEthane = .018;
-const double texasPipelinePropane = 0.004;
-const double texasPipelineCO2 = 0.004;
-const double texasPipelineButane = 0.0015;
-
-const double rockyPipelineMethane = 0.945;
-const double rockyPipelineEthane = 0.035;
-const double rockyPipelinePropane = 0.006;
-const double rockyPipelineCO2 = 0.0075;
-const double rockyPipelineButane = 0.003;
-
-const double peruvianLNGMethane = 0.883;
-const double peruvianLNGEthane = 0.105;
-const double peruvianLNGPropane = 0;
-const double peruvianLNGCO2 = 0;
-const double peruvianLNGButane = 0;
-
-const double highAssociated1Methane = 0.8365;
-const double highAssociated1Ethane = 0.1075;
-const double highAssociated1Propane = 0.027;
-const double highAssociated1CO2 = 0;
-const double highAssociated1Butane = 0.002;
-
-const double highAssociated2Methane = 0.875;
-const double highAssociated2Ethane = 0.045;
-const double highAssociated2Propane = 0.044;
-const double highAssociated2CO2 = 0;
-const double highAssociated2Butane = 0.012;
 
 void calculate_cp(int tempCnt, int pressureCnt, double &CPMethane, double &CPPropane, double &CPCO2, double &CPEthane, double &CPButane, double &thermConMethane, double &thermConEthane, double &thermConPropane, double &thermConCO2, double &thermConButane) {
```

Comment:
Let's change this to a boolean function. It'll be nicer to check for it in the future instead of looking through the outputs.